tests/custom.c

```diff
@@ -0,0 +1,25 @@
+/*
+ * Compiled with './build'
+ */
+
+#include "../elf/include/bpf.h"
+
+#define SEC(NAME) __attribute__((section(NAME), used))
+
+struct bpf_map_def SEC("maps/dummy_hash_custom") dummy_hash_custom = {
+	.type = BPF_MAP_TYPE_HASH,
+	.key_size = sizeof(int),
+	.value_size = sizeof(unsigned int),
+	.max_entries = 128,
+	.pinning = PIN_CUSTOM_NS,
+	.namespace = "fw"
+};
+
+struct bpf_map_def SEC("maps/dummy_array_custom") dummy_array_custom = {
```

Comment:
IMHO it's sufficient to test with one map `PIN_CUSTOM_NS` for gobpf matters and rely on the kernel to work with other maps types as well.

Reply:
i.e. remove the `testgroup2` test.